### DISCUSSION TITLE
Vue DevTools are not available

### DISCUSSION THREAD

**Discussion by ruslankonev** [nuxt/nuxt#18283] [Questions] [answered]
Is it true?

**Comment by ruslankonev** [comment, marked as answer]
Duplicate of https://github.com/nuxt/framework/discussions/1011

**Comment by Juninhoww2** [comment]
What is the answer?

**Reply by ruslankonev** [reply]
You need to use a Vue3 DevTools → https://github.com/nuxt/framework/discussions/1011#discussioncomment-1475975
In my case this extension was disabled, and did not work
